CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# dbt_fivetran_log version.version
+# dbt_fivetran_log v1.11.0
+[PR #141](https://github.com/fivetran/dbt_fivetran_log/pull/141) includes the following updates:
 
-## Documentation
+## Schema Changes: Adding the Transformation Runs Table
+- We have added the `transformation_runs` source table. Note that not all customers have the `transformation_runs` source table, particularly if they are not using Fivetran Transformations. Therefore, the `transformation_runs` table will only populate if the table exists in your schema, via a new variable `fivetran_platform_using_transformations`, which automatically checks for the table. If the table doesn't exist, the staging `stg_fivetran_platform__transformation_runs` model will persist as an empty model and respective downstream fields will be null. 
+
+- If the `transformation_runs` source table exists in your schema, `fivetran_platform_using_transformations` will be set to True and the following updates apply:
+  - Added a new staging [`stg_fivetran_platform__transformation_runs`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.stg_fivetran_platform__transformation_runs) model. 
+    - We have also added the `get_transformation_runs_columns()` macro to ensure all required columns are present.
+  - Added the following fields to the [`fivetran_platform__usage_mar_destination_history`](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__usage_mar_destination_history) end model for each destination and month:
+    - `paid_model_runs`
+    - `free_model_runs`
+    - `total_model_runs`
+  - If you would like to override these updates, you can also manually disable the `fivetran_platform_using_transformations` variable by setting it to False in your project.yml
```

Comment:
This CHANGELOG entry is confusing or misleading. I read this and imagine that if I define `fivetran_platform_using_transformations: false` then these three mentioned fields will not be present in the end model. However, setting this to false will simply ensure all of the results are `null`.

Can you confirm the `null` behavior is what is expected? If so, we should rethink this. Why would someone want to manually set this variable as `false` if the fields will still be present, just empty?

Reply:
Right it will be null if disabled. You're saying that it would be more likely that a customer disabling the variable would expect the fields not to be present at all, rather than null? The reason I did it this way was to mirror what is done for usage_cost, where if that table doesn't exist or is disabled, its downstream fields are all null. 

`fivetran_platform__usage_pricing: False`

![image](https://github.com/user-attachments/assets/2c9732e7-c67c-42d7-81c7-8a0ce89e637c)

Reply:
Updated the language to make it clearer that the fields will still be present but null.